simple_qa_rnn/data/SimpleQuestions_v2/FB-README.txt

```diff
@@ -0,0 +1,84 @@
+The SimpleQuestions Dataset
```

Comment:
do we really want to check in the dataset here? I thought we were leaving data here?
https://github.com/castorini/data/tree/master/SimpleQuestions_v2